core/controllers/admin.py

```diff
@@ -1066,3 +1066,30 @@ def get(self):
             'number_of_pending_deletion_models': (
                 wipeout_service.get_number_of_pending_deletion_requests())
         })
+
+
+class GetModelsRelatedToUserHandler(base.BaseHandler):
+    """Handler for getting all the models related to a user with specific ID."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        user_id = self.request.get('user_id', None)
+        if user_id is None:
+            raise self.InvalidInputException('Missing user_id param')
+        user_is_deleted = wipeout_service.verify_user_deleted(
+            user_id, include_delete_at_end_models=True)
+        self.render_json({'related_models_exist': not user_is_deleted})
+
+
+class DeleteUserHandler(base.BaseHandler):
+    """Handler for deleting a user with specific ID."""
+
+    @acl_decorators.can_access_admin_page
+    def delete(self):
```

Comment:
This is pretty dangerous, is there some way we can turn it off immediately after release (maybe with a planned immediate hotfix)?